src/controllers/controller.polarArea.js

```diff
@@ -138,6 +138,31 @@ module.exports = function(Chart) {
 			});
 		},
 
+		getStartAngle: function(customAngles, dataSetData, metaData, datasetStartAngle, circumference, index) {
+
+			// If there is NaN data before us, we need to calculate the starting angle correctly.
+			// We could be way more efficient here, but its unlikely that the polar area chart will have a lot of data
+
+			var previousAngle = 0;
+			var visibleCount = 0;
+
+			for (var i = 0; i < index; ++i) {
+				if (customAngles !== undefined) {
+					previousAngle += customAngles[visibleCount];
+				}
+				if (!isNaN(dataSetData[i]) && !metaData[i].hidden) {
+					++visibleCount;
+				}
+			}
+			var startAngle = 0;
```

Comment:
this project has a convention of declaring all the variables at the beginning of a function as it apparently improves code minimization. can you move the declaration of `startAngle` and also `i` to occur with `previousAngle` and `visibleCount`?